src/System.Net.Security/src/System/Net/SecurityStatusAdapterPal.Windows.cs

```diff
@@ -0,0 +1,82 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace System.Net
+{
+    internal static class SecurityStatusAdapterPal
+    {
```

Comment:
Nit: elsewhere when we've done static readonly dictionary lookup tables like this, we've added a debug-only static cctor assert that verifies it's being sized correctly, e.g.
```C#
private const int StatusDictionarySize = 39;

#if DEBUG
static SecurityStatusAdapterPal()
{
    Debug.Assert(s_statusDictionary.Count == StatusDictionarySize, $"Expected size {StatusDictionarySize}, got size {s_statusDictionary.Count}");
}
#endif

private static readonly ... = new BidirectionDictionary<...>(StatusDictionarySize);
```